README.md

```diff
@@ -34,29 +34,49 @@ def handler(event:, context:)
     JSON.generate(body)
 end
 
-SignalFx::Lambda::Tracing.register_handler(&method(:handler))
+SignalFx::Lambda.register_handler(metrics: true, tracing: true, &method(:handler))
 ```
 
 `register_handler` will accept any block.
 
+If passing in a block parameter, it must be the last argument.
+
+It also takes these optional arguments:
+- `metrics`: Enable reporting of metrics. Default: `true`
+- `tracing`: Enable tracing. Default: `true`
+
 ### Tracer configuration
 
 The tracer used by the function is configured through environment variables:
 
 ```
 SIGNALFX_ACCESS_TOKEN
-SIGNALFX_INGEST_URL
 SIGNALFX_SERVICE_NAME
+SIGNALFX_TRACING_URL
 ```
 
-In production, `SIGNALFX_INGEST_URL` should be pointing to your [Smart Gateway](https://docs.signalfx.com/en/latest/apm/apm-deployment/smart-gateway.html).
+In production, `SIGNALFX_TRACING_URL` should be pointing to your [Smart Gateway](https://docs.signalfx.com/en/latest/apm/apm-deployment/smart-gateway.html).
 When pointing to the Smart Gateway, an access token is not needed. When not
 configured, the ingest URL defaults to `https://ingest.signalfx.com/v1/trace`,
 which requires an access token to be configured.
 
 The tracer will be persisted across invocations to the same context, reducing
 the time needed for tracer initialization.
 
+### SignalFx client configuration
+
+The SignalFx client requires the following environment variables to be set:
+
+```
+SIGNALFX_ACCESS_TOKEN
+SIGNALFX_METRICS_URL
```

Comment:
What about `SIGNALFX_TRACING_URL`?

Reply:
This is just for configuring the metrics endpoint